src/ray/common/constants.h

```diff
@@ -18,7 +18,7 @@
 #include <stdint.h>
 
 /// Length of Ray full-length IDs in bytes.
-constexpr size_t kUniqueIDSize = 20;
+constexpr size_t kUniqueIDSize = 28;
```

Comment:
The reason why this was set to 20 was because it needed to be aligned with plasma object ID length. 
Do we not have this limit now?

Reply:
I think it might have been resolved when we merged plasma into Ray. It seems to work now. cc @suquark

Reply:
yes, Plasma uses the objectid of Ray now.